src/MuTalk-Tests/MTMatrixTest.class.st

```diff
@@ -0,0 +1,104 @@
+Class {
+	#name : 'MTMatrixTest',
+	#superclass : 'TestCase',
+	#instVars : [
+		'matrix',
+		'trivialMutant',
+		'equivalentMutants',
+		'includedMutants'
+	],
+	#category : 'MuTalk-Tests',
```

Comment:
I wonder if we shouldn't put this class and friends in a Mutalk-Utilities-Tests package?

Reply:
Yes it's probably better